tuya.cpp

```diff
@@ -1028,11 +1035,25 @@ void DeRestPluginPrivate::handleTuyaClusterIndication(const deCONZ::ApsDataIndic
                         }
                     }
                     break;
-                    case 0x0269: // Boost time
+                    case 0x0269: // Boost time in second or Heatpoint
                     {
+                        if (sensorNode->manufacturer().endsWith(QLatin1String("hn3negr")))
+                        {
+                            qint16 temp = static_cast<qint16>(data & 0xFFFF) * 100 / 2;
+   
+                            ResourceItem *item = sensorNode->item(RConfigHeatSetpoint);
+
+                            if (item && item->toNumber() != temp)
+                            {
+                                item->setValue(temp);
+                                Event e(RSensors, RConfigHeatSetpoint, sensorNode->id(), item);
+                                enqueueEvent(e);
```

Comment:
`enqueueEvent(Event(RSensors, RConfigHeatSetpoint, sensorNode->id(), item));`

I think we can use the shorter version here.

Reply:
done